### ISSUE TITLE
Changelog?

### ISSUE THREAD

**Issue by ForsakenHarmony** [bojand/mailgun-js#188]
Where's your changelog?

**Comment by bojand** [comment]
Hello, unfortunately there isn't really one besides the commit log. Most changes for a while have been either minor bug fixes or documentation. You can see a list of all releases in the [releases](https://github.com/bojand/mailgun-js/releases) page and then inspect diffs by comparing associated tags. for example the [change in the last release](https://github.com/bojand/mailgun-js/compare/v0.14.0...v0.14.1) compared to previous one.

**Comment by ForsakenHarmony** [comment]
yeah, but I need a general overview of new features / breaking changes looking at commits for every package individually just takes too long

**Comment by ForsakenHarmony** [comment]
https://github.com/zeit/release

**Comment by dsernst** [comment]
A changelog would be very valuable to check for breaking changes when upgrading this package.
